lisp/ess-inf.el

```diff
@@ -46,6 +46,8 @@
 (require 'format-spec)
 (require 'ess-tracebug)
 
+(declare-function tramp-sh-handle-expand-file-name "tramp-sh")
```

Comment:
Should there be instead `(require 'tramp-sh)` inside of `ess-set-working-directory`?

Reply:
I don't think it is, since tramp is built in with Emacs we can assume it's available, unlike auto-complete. I've removed it and added `(require 'tramp-sh)` in that function.

Reply:
Would be good to investigate if that action could be done on remotes without using internal tramp functions.